### ISSUE TITLE
Update beta doesn't seem to work

### ISSUE THREAD

**Issue by haf** [fsprojects/Paket#793]
I have something like this:

paket.dependencies:

```
source ...
nuget A >= 0.2.0-beta1
nuget A.B >= 0.2.0-beta1
```

Where A.B depends on A.

Now, trying to do `mono paket.exe update nuget A` doesn't change a thing. I haven't investigated whether this is because it's beta or because `A.B` depends on `beta1`.

------------

Also:

```
source ..
nuget A > 0.2.0-beta1
nuget A.C
```

Gives error message (both are beta1):

```
Paket failed with:
	Could not find compatible versions for top level dependency:
     "Crypto  (from /paket.dependencies)"
   Available versions:
     - 0.2.0-beta2
   Try to relax the dependency or allow prereleases.
```

**Comment by forki** [comment]
I can't repro. I tested with:

    source https://www.nuget.org/api/v2
    
    nuget FsUnit >= 1.3.0.0
    nuget NUnit >= 2.6.3

and paket.lock:

    NUGET
      remote: https://www.nuget.org/api/v2
      specs:
        FsUnit (1.3.0.0)
          NUnit (>= 2.6.3)
        NUnit (2.6.3)

then running `paket update nuget FsUnit` gives:


    NUGET
      remote: https://www.nuget.org/api/v2
      specs:
        FsUnit (1.3.0.1)
          NUnit (>= 2.6.3)
        NUnit (2.6.3)

**Comment by haf** [comment]
I think it's only a problem with versions with 'special'/'prerelease' components

**Comment by forki** [comment]
ok. so we need to find public versions of such a constellation

**Comment by haf** [comment]
Works with latest - just tested. You can use EventStore.Client.FSharp for future reference ;)

**Comment by haf** [comment]
It seems that if I mention the package explicitly it upgrades, if there are no dependencies on it from other packages, but if I don't, and just do `paket.update`; `Try to relax the dependency or allow prereleases.` and the latest version isn't even listed in the list of 'available versions'.

**Comment by haf** [comment]
`["0.2.0-beta9","0.2.0-beta8","0.2.0-beta7","0.2.0-beta6","0.2.0-beta5","0.2.0-beta4","0.2.0-beta2","0.2.0-beta1","0.1.0-beta1"]` are the available versions, but 'available versions' only lists 0.2.0-beta8

**Comment by forki** [comment]
I just found that we don' respect prerelease settings in this code path. Will try to add this.
